### PR TITLE
Add AWS to adopters list

### PR DESCRIPTION
**Pull request by gouthamve**
https://aws.amazon.com/prometheus/faqs/ mentions that Cortex powers the service. I'm curious if you'd be open to being listed as an adopter @ranton256

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 This is the list of organisations that are using Cortex in **production environments** to power their metrics and monitoring systems. Please send PRs to add or remove organisations.
 
+* [Amazon Web Services (AWS)](https://aws.amazon.com/)
```
Comment (ranton256) [inline ADOPTERS.md:5]: Would it make more sense to link to https://aws.amazon.com/prometheus/ instead?

**Review by ranton256 — APPROVED** [review]
LGTM
I was going to send this anyway, but you were faster.

File: ADOPTERS.md (hunk 1 above)
Comment (gouthamve) [reply ADOPTERS.md]: done!

**Review by pstibrany — APPROVED** [review]

**Comment by alolita** [comment]
lgtm.

**Pull request merged**